CMakeLists.txt

```diff
@@ -47,26 +85,38 @@ if(NOT CUKE_DISABLE_BOOST_TEST)
 endif()
 
 if(CUKE_USE_STATIC_BOOST)
+    message(STATUS "Using STATIC BOOST")
     set(CUKE_STATIC_BOOST_LIBS ${CUKE_CORE_BOOST_LIBS})
     # "An external test runner utility is required to link with dynamic library" (Boost User's Guide)
-    set(CUKE_DYNAMIC_BOOST_LIBS ${CUKE_TEST_BOOST_LIBS})
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBOOST_TEST_DYN_LINK")
+    if(NOT CUKE_DISABLE_BOOST_TEST)
+        set(CUKE_DYNAMIC_BOOST_LIBS ${CUKE_TEST_BOOST_LIBS})
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBOOST_TEST_DYN_LINK")
+    endif()
 
     if(NOT MSVC)
         find_package(Threads)
         set(CUKE_EXTRA_LIBRARIES ${CUKE_EXTRA_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
     endif()
 else()
+    message(STATUS "Using DYNAMIC BOOST")
     set(CUKE_DYNAMIC_BOOST_LIBS ${CUKE_CORE_BOOST_LIBS} ${CUKE_TEST_BOOST_LIBS})
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBOOST_ALL_DYN_LINK")
 endif()
 
 if(CUKE_STATIC_BOOST_LIBS)
+    if(VERBOSE)
+        message(STATUS "Using STATIC BOOST libs: ${CUKE_STATIC_BOOST_LIBS}")
+    endif()
     set(Boost_USE_STATIC_LIBS ON)
+    set(Boost_USE_STATIC_RUNTIME ON)
+    set (Boost_MULTITHREADED ON) 
```

Comment:
Yes - on windows only way I could get it running was to statically link everything